src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -819,39 +819,36 @@ impl<T: AstInfo> AstDisplay for CreateConnectionOption<T> {
 impl_display_t!(CreateConnectionOption);
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub enum KafkaConfigOptionName {
-    CompressionType,
+pub enum KafkaSourceConfigOptionName {
```

Comment:
🙇 –– I've been meaning to fix this for weeks and just couldn't get around to it. tysm.